languages/elixir/exercises/concept/booleans/test/rules_test.exs

```diff
@@ -1,57 +1,71 @@
 defmodule RulesTest do
   use ExUnit.Case
 
-  test "if ghost gets eaten" do
-    assert Rules.eat_ghost?(true, true)
-  end
+  describe "eat_ghost?/2" do
```

Comment:
I added describes to group up the functions under test, plus changed wording and reworked some cases. Let me know what you think.